website/case-studies/implementation-of-flint-modules.md

```diff
@@ -0,0 +1,407 @@
+
+```
+AGC = * [ 1 -  e ^ { -k *Age } ] ^ { ( 1 / (1-m) }
+```
+
+-   **AGC**: Above-ground stand carbon (Tonnes carbon per hectare).
+-   **Max**: Asymptote – maximum peak carbon yield (Tonnes carbon per hectare).
+-   **k**: Parameter used in modelling tree growth (dimensionless).
+-   **Age**: Age of the forest (years).
+-   **m**: Parameter used in modelling tree growth (dimensionless).
+
+This module provides us the estimation of Aboveground biomass on the basis of the age of forests. Belowground biomass is calculated as a ratio from aboveground biomass. This equation is also used within the Forest Growth module. There is a separate repository for the [Chapman Richards module](https://github.com/moja-global/FLINT.Module.Chapman_Richards). It works on a monthly time step.
+
+### Input
+
+-   Time Step (Monthly)
+-   Dead Organic Matter
+-   Soil organic matter
+-   Pool 1 : Non-CO2 gases (The selection of carbon pools or non-CO2 gases for estimation will depend on the significance of the pool and tier selected for each land-use category)
+-   Forest Age
+-   Forest Type
+-   Forest Cover
+-   Soil Carbon
+-   Spatial Location of Forest
+
+### Output
+
+-   Above-ground biomass
+-   Below-ground biomass
+
+For Forest Land complete documentation refer to the [Forest land documentation](https://www.ipcc-nggip.iges.or.jp/public/2006gl/pdf/4_Volume4/V4_04_Ch4_Forest_Land.pdf) which is a part of Volume 4: Agriculture, Forestry and Other Land Use.
+
+## GCBM Module
+
+The GCBM (Generic Carbon Budget Model) is a combination of the FLINT platform with the science modules developed by the Canadian Forest Service (CFS).
+
+These are the same science modules used in the first generation tool ([CBM-CFS3](https://www.nrcan.gc.ca/climate-change/impacts-adaptations/climate-change-impacts-forests/carbon-accounting/carbon-budget-model/cbm-cfs3/13089)). Since the science and processes behind both tools are very similar, it is relatively easy to transition from CBM-CFS3 to GCBM.
+
+The CBM-CFS3 is widely used throughout Canada and globally and its use is supported by the Canadian Forest Service of Natural Resources Canada.
+
+The next generation GCBM is currently used by the CFS with a number of partner organizations to advance the science of forest carbon estimation and to support policy analyses such as the assessment of mitigation options in the forest sector.
+
+In GCBM, forest inventory, and disturbance events are spatially explicit. The location of disturbances are explicit in spatial layers instead of being rule based. The GCBM Module is able to simulate large landscapes easily.
+
+|![Diagram depicting the high-level overview of GCBM.](../static/img/gcbm-overview-module-case-study.png) |
+|:--:|
+| Diagram depicting the high-level overview of GCBM. |
```

Comment:
Only the very middle box of this diagram relates to modules. Perhaps you can crop it?